taskt/Core/Utilities/CommonUtilities/VariableMethods.cs

```diff
@@ -209,14 +201,12 @@ public static string ConvertToUserVariable(this String str, object sender)
                         {
                             str = str.Replace(potentialVariable, (string)varCheck.GetDisplayValue());
                         }
-                        catch (Exception)
+                        catch (Exception ex)
                         {
-
+                            throw new Exception(ex.Message);
```

Comment:
Is there any point to this catch if it's just re-throwing the exception?

Reply:
It's just re-throwing exception.
Updated it.